iml-manager-client/src/lib.rs

```diff
@@ -2,16 +2,21 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-use futures::{Future, IntoFuture as _};
-use reqwest::{header, r#async::Client, Url};
+use futures::{Future, IntoFuture as _, Stream as _};
```

Comment:
These changes should not live in this patch.

Reply:
I had rebased on these changes earlier but changed it back to master. Not sure why it's in here.